main.py

```diff
@@ -1,10 +1,38 @@
 import init_django_orm  # noqa: F401
-
+import json
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players_data = json.load(file)
+    for nickname, player_info in players_data.items():
+        race = Race.objects.get_or_create(
+            name=player_info["race"]["name"],
+            defaults={"description": player_info["race"]["description"]}
+        )[0]
```

Comment:
```suggestion
        race, _ = Race.objects.get_or_create(
            name=player_info["race"]["name"],
            defaults={"description": player_info["race"]["description"]}
        )
```
For me this one looks a bit more readable